src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -653,6 +659,25 @@ else if (saturation > AVERAGE_SATURATION)
         }
     }
 
+    /**
+     * repair random equipped/held item with mending enchant.
+     *
+     * @param xp amount of xp available to mend with
+     * @return xp left after mending
+     */
+    private double applyMending(double xp) {
+        ItemStack tool = EnchantmentHelper.getEnchantedItem(Enchantments.MENDING, this);
+
+        if (tool != null && tool.isItemDamaged()) {
+            //2 xp to heal 1 dmg
+            double dmgHealed = Math.min(xp / 2, tool.getItemDamage());
+            xp -= dmgHealed * 2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "xp". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1226)